agent/interface.js

```diff
@@ -51,12 +51,28 @@ function createCommandDefinitions(agent) {
         summary: Flags.string({
           description: "Specify output file for summarize results",
         }),
+        prerun: Flags.string({
```

Comment:
I'm fine with not documenting this as an experimental feature until we know if it's stable or not.